src/web/urls.py

```diff
@@ -2,7 +2,10 @@
 
 from django.urls import path
 
+app_name = "web"
 urlpatterns = [
     path('', views.index, name="index"),
     path('events', views.events, name="events"),
+    path('groups', views.list_tech_groups, name="list_tech_groups"),
+    path('groups/<int:pk>', views.get_tech_group, name="get_tech_group"),
```

Comment:
you should use "group" for the detail view not "groups" as the page will explicitly display a single group. Also, personal preference and pro tip, keeping the path and the name the same will save you headaches down the line (totally optional suggestion).

Reply:
Left groups via in person discussion